src/main/java/jenkins/advancedqueue/PriorityConfigurationPlaceholderTaskHelper.java

```diff
@@ -0,0 +1,29 @@
+package jenkins.advancedqueue;
+
+
+import hudson.model.Job;
+import hudson.model.Queue;
+import jenkins.advancedqueue.sorter.ItemInfo;
+import jenkins.advancedqueue.sorter.QueueItemCache;
+import jenkins.model.Jenkins;
+import org.jenkinsci.plugins.workflow.support.steps.ExecutorStepExecution;
+
+class PriorityConfigurationPlaceholderTaskHelper {
+
+    boolean isPlaceholderTask(Queue.Task task) {
+        return task instanceof  ExecutorStepExecution.PlaceholderTask;
```

Comment:
I would add a isPlaceholderTaskUsed() check before invoking instanceOf. Or just check class name by a string comparison. Both options should prevent exception here in the case of API misusage